src/suite.py

```diff
@@ -4,7 +4,7 @@
 from text_modifier import *

```

Comment:
This is optional, but I think style guides (e.g. http://google.github.io/styleguide/pyguide.html) typically encourage using full package names. (This is definitely the case for Java at Google, maybe for python, too.) Since you didn't create this file, you generally wouldn't be held responsible for cleaning this up.